crates/driver/src/tests/setup/blockchain.rs

```diff
@@ -602,11 +778,11 @@ impl Blockchain {
     /// fulfill the orders.
     pub async fn fulfill(
         &self,
-        orders: impl Iterator<Item = &Order>,
+        orders: impl Iterator<Item = &(Order, bool)>,
```

Comment:
Adding this `is_jit_order` flag seems a bit shoehorned in. Since it only changes whether the generated order gets put into a `JIT` or a regular order maybe the call could be made responsible for wrapping the orders into the expected types? 🤔 
Not sure about that, though.